### ISSUE TITLE
allow to automatically send fog updates

### ISSUE THREAD

**Issue by n1ru4l** [dungeon-revealer/dungeon-revealer#1365]
it should be possible to opt in to automatically sending fog updates to the connected players.

**Comment by thebreuer** [comment]
Is it still up? I would like to take a look at it.

**Comment by n1ru4l** [comment]
@thebreuer Yes this is still something that needs to be taken care of and I would appreciate any help :)

The idea is that the DM can change a setting (simple checkbox for opting into automatic fog updates sending). I am not 100% sure where we should place that option.

Here are some ideas:

- In the chat settings
- Upon doing a long-press on the Send button
- Upon hovering over the send button
- as a new setting modal that is opened via a new button next to the note search and toggle chat visibility window (this would allow adding more setting options 🤔 )

**Comment by thebreuer** [comment]
Awesome :+1: 
I think a new settings modal would be neat for further extensibility; I will drill into it
